pkg/entities/feedback.go

```diff
@@ -11,5 +15,52 @@ func (e *Entity) HandleUserFeedback(req *request.UserFeedbackRequest) error {
 		e.log.Fields(logger.Fields{"req": req}).Error(err, "[entity.HandleUserFeedback] e.svc.Discord.SendFeedback failed")
 		return err
 	}
+
+	// store feedback
+	err = e.repo.UserFeedback.CreateOne(&model.UserFeedback{
+		DiscordID: req.DiscordID,
+		Command:   req.Command,
+		Feedback:  req.Feedback,
+		Status:    "none",
+	})
+	if err != nil {
+		e.log.Fields(logger.Fields{"req": req}).Error(err, "[entity.HandleUserFeedback] e.repo.UserFeedback.CreateOne failed")
+		return err
+	}
 	return nil
 }
+
+func (e *Entity) UpdateUserFeedback(req *request.UpdateUserFeedbackRequest) error {
+	err := e.repo.UserFeedback.UpdateStatusByID(req.ID, req.Status)
+	if err != nil {
+		e.log.Fields(logger.Fields{"req": req}).Error(err, "[entity.UpdateUserFeedback] e.repo.UserFeedback.UpdateStatusByID failed")
+		return err
+	}
+	return nil
+}
+
+func (e *Entity) GetAllUserFeedback(filter string, value string) (*response.UserFeedbackResponse, error) {
```

Comment:
Since we wrap data in handler, change return type
```func (e *Entity) GetAllUserFeedback(filter string, value string) (feedbacks []model.UserFeedback, err error) ```